dev/release.py

```diff
@@ -1,12 +1,18 @@
 #!/usr/bin/env python
 import click
+from datetime import datetime
 from six.moves import input
 from subprocess import check_call, check_output
 import sys
 
-WORKING_BRANCH = "PREP_RELEASE_%s"
 DATABRICKS_REMOTE = "git@github.com:databricks/spark-deep-learning.git"
-PUBLISH_MODES = {"local": "publishLocal", "m2": "publishM2", "spark-package": "spDist"}
+PUBLISH_MODES = {"local": "publishLocal", "m2": "publishM2", "spark-package-publish": "spPublish"}
```

Comment:
I still don't know why the changes are relevant to "build and push docs". If they are necessary to include, it is worth an inline GitHub comment here. If they are irrelevant, we should create a separate PR.

Keeping PRs minimal is to help both PR review and merge. For example, I spent ~10 mins yesterday trying to understand how it is relevant to doc build but got no clue.

Reply:
This isn't related to docs, I should have commented sooner.

I realized while working on docs that we'll need this publish method available to use this for an actual release. I can split this into another PR if you think we should.